templates/widgets/teachers.html

```diff
@@ -1,4 +1,4 @@
 <div id="teachers_banner">
-  <object type="application/x-shockwave-flash" data="img/summer_banner.swf" height="170" width="287"></object>
+  <img src='/img/knowledge_day.gif' height="170" width="287"></object>
```

Comment:
First, let's stick with double quotes for attribute values. Second, why have you left the closing `object` tag? Third, image size isn't proportional now, since its original size is 280x190. So, just leave off `height` value.